apple/Sources/FerrostarCore/FerrostarCore.swift

```diff
@@ -359,8 +360,9 @@ extension FerrostarCore: LocationManagingDelegate {
         update(newState: newState, location: location)
     }
 
-    public func locationManager(_: LocationProviding, didUpdateHeading newHeading: Heading) {
-        state?.heading = newHeading
+    public func locationManager(_: LocationProviding, didUpdateHeading _: Heading) {
+        // TODO: Make use of heading in TripState?
```

Comment:
I suspect we probably want this to be attached to the UserLocation -> Course somehow? But we should discuss & consider a new ticket. Don't think it was actually doing anything.

Reply:
https://github.com/stadiamaps/ferrostar/issues/169